src/theme/default.less

```
@@ -134,6 +200,128 @@ mark {
+.ant-table-column-sorter {
+  .ant-table-column-sorter-inner {
+    flex-direction: row;
+    align-items: center;
+    gap: 2px;
+  }
+
+  .ant-table-column-sorter-up,
+  .ant-table-column-sorter-down {
+    width: 12px;
+    height: 12px;
+    background-color: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)